tests/unittests/UIKit/NSParagraphStyleTests.mm

```diff
@@ -0,0 +1,65 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+#import <Foundation/Foundation.h>
+#import <CoreText/CTParagraphStyle.h>
+#import <UIKit/NSMutableParagraphStyle.h>
+#import "NSParagraphStyleInternal.h"
+#import "Starboard/SmartTypes.h"
+
+TEST(NSParagraphStyle, ShouldConvertToCTParagraphStyle) {
+    NSMutableParagraphStyle* style = [[NSMutableParagraphStyle new] autorelease];
```

Comment:
>NSMutableParagraphStyle [](start = 39, length = 23)

Please add a test that copies can/cannot be mutated as expected on the reference platform.